config/default/views.view.content_orphaned.yml

```diff
@@ -0,0 +1,632 @@
+                  hs_news: '0'
+                  hs_person: '0'
+                  hs_private_page: '0'
+                  hs_publications: '0'
+                  hs_research: '0'
+              3:
+                title: 'Private Page'
+                operator: in
+                value:
+                  hs_private_page: hs_private_page
+                  all: '0'
+                  hs_course: '0'
+                  hs_event: '0'
+                  hs_event_series: '0'
+                  hs_basic_page: '0'
+                  hs_news: '0'
+                  hs_person: '0'
+                  hs_publications: '0'
+                  hs_research: '0'
+      filter_groups:
+        operator: AND
+        groups:
+          1: AND
+      style:
+        type: table
+        options:
+          grouping: {  }
+          row_class: ''
+          default_row_class: true
+          columns:
+            title: title
+            type: type
+            changed: changed
+            operations: operations
+          default: changed
+          info:
+            title:
+              sortable: true
+              default_sort_order: asc
+              align: ''
+              separator: ''
+              empty_column: false
+              responsive: ''
+            type:
+              sortable: true
+              default_sort_order: asc
+              align: ''
+              separator: ''
+              empty_column: false
+              responsive: ''
+            changed:
+              sortable: true
+              default_sort_order: desc
+              align: ''
+              separator: ''
+              empty_column: false
+              responsive: ''
+            operations:
+              align: ''
+              separator: ''
+              empty_column: false
+              responsive: ''
+          override: true
+          sticky: false
+          summary: ''
+          empty_table: false
+          caption: ''
+          description: ''
+      row:
+        type: fields
+        options:
+          default_field_elements: true
+          inline: {  }
+          separator: ''
+          hide_empty: false
+      query:
+        type: views_query
+        options:
+          query_comment: ''
+          disable_sql_rewrite: false
+          distinct: false
+          replica: false
+          query_tags: {  }
+      relationships: {  }
+      header: {  }
+      footer: {  }
+      display_extenders: {  }
+    cache_metadata:
+      max-age: -1
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url
+        - url.query_args
+        - 'user.node_grants:view'
+        - user.permissions
+      tags: {  }
+  page_1:
+    id: page_1
+    display_title: Page
+    display_plugin: page
+    position: 1
+    display_options:
+      display_extenders: {  }
+      path: admin/content/orphaned-pages
+      menu:
+        type: tab
+        title: Orphaned
```

Comment:
According to [the ticket](https://app.clickup.com/t/36718269/SHS-5838), this should be "Orphaned Pages".